package.json

```diff
@@ -1,51 +1,30 @@
 {
   "name": "cnn-logger",
-  "version": "1.5.0",
-  "description": "CNN Logger",
-  "main": "lib/logger.js",
-  "directories": {
-    "example": "./example",
-    "lib": "./lib",
-    "test": "./test"
-  },
+  "version": "2.0.0",
```

Comment:
Do not change the version just yet. We will bump once we merge into master.

Reply:
I will change that back